test/cases/ostree-ng.sh

```diff
@@ -127,11 +127,12 @@ build_image() {
             }
         }" | tee "$COMPOSE_START"
     else
+        # Test ref begining with /
         sudo curl --silent --header "Content-Type: application/json" --unix-socket /run/weldr/api.socket http://localhost/api/v1/compose --data "{
             \"blueprint_name\": \"$blueprint_name\",
             \"compose_type\": \"$image_type\",
             \"ostree\": {
-                \"ref\": \"$OSTREE_REF\"
+                \"ref\": \"/$OSTREE_REF\"
```

Comment:
@achilleas-k could you please remove a `/` before `$OSTREE_REF here when you push your fix? Thanks.

Reply:
Will do!